README.md

```diff
@@ -67,9 +37,7 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- We thank microverse for the opportunity to work on this project
 
 ## 📝 License
```

Comment:
- [ ] As highlighted by the previous reviewer, please update your README with the missing sections especially the LIcence section which is currently displaying an `error 404 page`. Kindly fix it please.

Reply:
This issue is well resolved! ✔️